docs/modules/ROOT/pages/mutations/create.adoc

```diff
@@ -97,35 +97,36 @@ mutation {
 
 This will create a User with name "John Doe", an introductory Post, both of which will be returned with their autogenerated IDs.
 
-== `connectOrCreate` relations
-If a related node has a `@unique` or `@id` directive defined, `connectOrCreate` can be used in a nested create to perform a `MERGE` operation on the related node, creating a new relation and the related node if it doesn't exists.
+== `connectOrCreate` relationships
+If a related node has a `@unique` or `@id` directive defined, `connectOrCreate` can be used in a nested create to perform a `MERGE` operation on the related node, creating a new relationship and the related node if it doesn't exist.
+
+Consider the following type definitions:
 
-Consider the following type definition:
 [source, graphql, indent=0]
 ----
 type Actor {
     name: String!
-    movies: [Movie] @relationship(type: "ACTED_IN", direction: OUT)
+    movies: [Movie!]! @relationship(type: "ACTED_IN", direction: OUT)
```

Comment:
It's not here. Found one more of the Many-to-* relationships typeDef updates:
https://github.com/neo4j/graphql/blob/dev/docs/modules/ROOT/pages/type-definitions/cypher.adoc line 189